ndk_compat/cpu-features.c

```diff
@@ -128,11 +128,11 @@ static void android_cpuInit(void) {
   if (info.features.sse4_2) g_cpuFeatures |= ANDROID_CPU_X86_FEATURE_SSE4_2;
   if (info.features.avx) g_cpuFeatures |= ANDROID_CPU_X86_FEATURE_AVX;
   if (info.features.avx2) g_cpuFeatures |= ANDROID_CPU_X86_FEATURE_AVX2;
-    // if (info.features.) g_cpuFeatures |= ANDROID_CPU_X86_FEATURE_MOVBE;
-    // if (info.features.) g_cpuFeatures |= ANDROID_CPU_X86_FEATURE_AES_NI;
-    // if (info.features.) g_cpuFeatures |= ANDROID_CPU_X86_FEATURE_RDRAND;
-    // if (info.features.) g_cpuFeatures |= ANDROID_CPU_X86_FEATURE_POPCNT;
-    // if (info.features.) g_cpuFeatures |= ANDROID_CPU_X86_FEATURE_SHA_NI;
+  if (info.features.movbe) g_cpuFeatures |= ANDROID_CPU_X86_FEATURE_MOVBE;
```

Comment:
nit: I would have organised this check in the same order than the `X86FeaturesEnum` list.
i.e. `aes` at top then here: `sha`, `popcnt`, `movbe`, `rdrand`

Reply:
I reordered to use the same order as in the NDK.